components.json

```
@@ -266,7 +270,7 @@
 				"oscript": "OneScript"
 			},
 			"owner": "Diversus23"
-		},
+		},		
```

Please remove this tab.
```suggestion
		},
```